server/src/main/java/org/tbbtalent/server/service/db/impl/BrandingServiceImpl.java

```diff
@@ -70,8 +74,8 @@ public BrandingInfo getBrandingInfo(@Nullable String partnerAbbreviation) {
     private @NotNull BrandingInfo extractBrandingInfoFromPartner(@NonNull Partner partner) {
         BrandingInfo info = new BrandingInfo();
         info.setLogo(partner.getLogo());
-        if (partner instanceof SourcePartner) {
-            info.setLandingPage(((SourcePartner)partner).getRegistrationLandingPage());
+        if (partner instanceof SourcePartner sourcePartner) {
+            info.setLandingPage((sourcePartner).getRegistrationLandingPage());
```

Comment:
Nice! I didn't know this syntax.

But do you need the (sourcePartner). This would work, right?

```suggestion
            info.setLandingPage(sourcePartner.getRegistrationLandingPage());
```